src/main/java/at/hannibal2/skyhanni/config/features/inventory/customwardrobe/ColorConfig.java

```diff
@@ -41,4 +43,7 @@ public class ColorConfig {
     @ConfigEditorColour
     public String bottomBorderColor = "0:255:255:0:0";
 
+    @ConfigOption(name = "Reset to Default", desc = "Reset all custom wardrobe color settings to the default.")
```

Comment:
This might just be me, but I feel like these could be moved to the top of the accordions so people actually know these exist
Its a lot of scrolling right now

Reply:
> This might just be me, but I feel like these could be moved to the top of the accordions so people actually know these exist Its a lot of scrolling right now

i put it at the bottom since the chroma reset button was almost near the bottom, but i do agree it should be something that is noticed quickly, especially when there's a lot of options to scroll through so i'll move it in a bit